### DISCUSSION TITLE
PnP Search Audience Targeting

### DISCUSSION THREAD

**Discussion by buszi2137** [microsoft-search/pnp-modern-search#2738] [General]
Hi, I'm new to PnP Search. I need help with audience targeting in this webpart. I want to search by this webpart site pages with audience targeting. I mean if users group is in audience targeting then show if it's not the dont show. Problem is when I switch on option Audience Targeting it doesn't change. I have switched on Audience Targeting in Site Pages library. On some pages i set groups where I'm not in.
Configuration of PnP Search Webpart
<img width="221" alt="image" src="https://user-images.githubusercontent.com/124353997/216574888-67e468d1-12d9-47a4-84d6-a5f89adc73ee.png">
<img width="223" alt="image" src="https://user-images.githubusercontent.com/124353997/216574955-4a05595b-bf31-47e9-b9ad-a2fd075b0776.png">
<img width="220" alt="image" src="https://user-images.githubusercontent.com/124353997/216575052-6480dfa3-d7f8-4e38-9c84-a7ecc0922f29.png">

Results it returns with audience targeting on in webpart
<img width="548" alt="image" src="https://user-images.githubusercontent.com/124353997/216575184-f0933900-7d74-4f49-8c98-a4e5a11f543f.png">
And withoout this options
<img width="547" alt="image" src="https://user-images.githubusercontent.com/124353997/216575308-57f633b1-a7cc-44dc-a2d1-1f2cf7f7c0a7.png">
Do you know how to do it ?
Best Regads
Kamil

**Comment by Parmstig** [comment]
Check the console log in your browser, in rare cases i have seen that changes you made in the properties pane are not beeing set.
You can also try and append this to your query (ModernAudienceAadObjectIds:{User.Audiences} OR NOT IsAudienceTargeted:true)
As is my understanding is does the exact same as activating Audiences.
